pattern-1/tile/tile.yml

```diff
@@ -96,6 +121,12 @@ packages:
           query: (( .properties.carbon_db_query.value ))
           username: (( .properties.carbon_db_username.value ))
           password: (( .properties.carbon_db_password.value ))
+        um_db:
+          jdbc_url: (( .properties.carbon_jdbc_url.value ))
```

Comment:
Instead of using `carbon_db*`, shall we use something similar to `reg_db*` as these properties are applied to the Registry DB?
Also, `carbon_jdbc_url` should be changed to  `carbon_db_jdbc_url`